src/Compilers/CSharp/Portable/Symbols/Source/ThisParameterSymbol.cs

```diff
@@ -40,10 +40,22 @@ public override RefKind RefKind
         {
             get
             {
-                return
-                    ((object)_containingType == null || _containingType.TypeKind != TypeKind.Struct) ? RefKind.None :
-                    ((object)_containingMethod != null && _containingMethod.MethodKind == MethodKind.Constructor) ? RefKind.Out :
-                    RefKind.Ref;
+                if (ContainingType?.TypeKind != TypeKind.Struct)
+                {
+                    return RefKind.None;
+                }
+
+                if (_containingMethod?.MethodKind == MethodKind.Constructor)
```

Comment:
Good point. I ended up not using the refKind of "this", so I am not sure what code actually uses this. I will check if we actually need this change.

---
In reply to: [118552148](https://github.com/dotnet/roslyn/pull/19713#discussion_r118552148) [](ancestors = 118552148)